rosdep/base.yaml

```diff
@@ -5753,6 +5753,16 @@ libsoundio-dev:
   macports: [libsoundio]
   nixos: [libsoundio]
   ubuntu: [libsoundio-dev]
+libsoup2.4-dev:
```

Comment:
I know that our typical policy is to make the key match the Ubuntu package name, but in this case I don't think it makes sense.  In particular, `libsoup-devel` on Fedora, for instance, is already libsoup 2.74.  So I'm going to suggest we call this key `libsoup-dev` instead.

Reply:
Sounds resonable. I have renamed it.

Reply:
@clalancette a downside of this is we now have a versionless key `libsoup-dev` and have to choose if or when  we'll upgrade it to the [libsoup-3.0-dev package](https://packages.debian.org/search?keywords=libsoup-3.0-dev).